### DISCUSSION TITLE
keyword extraction model not performing well after quantization using ONNX

### DISCUSSION THREAD

**Discussion by amriteshbytesview** [onnx/onnx#5110] [Q&A]
I have some question regarding the keyword extraction model which I have built using SimpleT5.
When I convert the model to ONNX using fastT5, the model output is affected very badly.  As when I predicted from the normal model for a sentence it give me 15 keywords while onnx model give only 2 keywords.
My question are:-
1. What could be the reason for such performance.
2. Is there is another method to reduce the inference time of the model.

Please help me over this..😶 
Thank you...

**Comment by xadupre** [comment]
How did you quantize your model? Are you using your model on CPU, GPU?
